docs/building/windows-full.md

```diff
@@ -70,7 +70,7 @@ What can you do with the produced binaries?
 
 **Important**: "We don’t support production deployments of these binaries on any platform". For PowerShell .NET (aka: FullCLR PowerShell) our recommendation is to continue using the PowerShell .NET version already shipping in Windows Client and Windows Server.
 
-The primary reason to build the PowerShell FullCLR binaries is to test backward compatibility, and interoperability between .NET and CoreCLR.  It is also important to mention some features like PowerShell Workflows are not currently available in the CoreCLR version. So we want to provide the ability for the Community to test CoreCLR PowerShell code changes while validating that these changes don't introduce regressions in .NET PowerShell (aka: as FullCLR PowerShell)
```

Comment:
One sentence per line is a wonderful thing in principle, but it's kinda a mess to do it piecemeal. It'd be much nicer if someone (possibly me) were to go through and just fix a whole bunch of files w/o doing anything else to them in that commit...

I really don't want to grow these commits too much... There's already a kitchen-sink waiting...

I'll do it for the blobs I'm changing in this PR, but I will probably refuse for the other PRs.